eng/Microsoft.Extensions.targets

```diff
@@ -3,24 +3,47 @@
     <_UseNet6Packages Condition=" '$(TargetFramework.Contains(net6.0))' == 'true' or '$(PackageType)' == 'DotnetPlatform' ">true</_UseNet6Packages>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(_UseNet6Packages)' == 'true' ">
-    <_MicrosoftHostingVersion>6.0.0-preview.5.21301.5</_MicrosoftHostingVersion>
-    <_MicrosoftDependencyInjectionVersion>$(_MicrosoftHostingVersion)</_MicrosoftDependencyInjectionVersion>
-    <_MicrosoftFileProvidersVersion>6.0.0-preview.5.21301.5</_MicrosoftFileProvidersVersion>
-    <_MicrosoftSystemCodeDomVersion>6.0.0-preview.5.21301.5</_MicrosoftSystemCodeDomVersion>
+    <_MicrosoftSystemCodeDomVersion>$(SystemCodeDomPackageVersion)</_MicrosoftSystemCodeDomVersion>
+    <_MicrosoftExtensionsHostingAbstractionsPackageVersion>$(MicrosoftExtensionsHostingAbstractionsPackageVersion)</_MicrosoftExtensionsHostingAbstractionsPackageVersion>
+    <_MicrosoftExtensionsDependencyInjectionAbstractionsPackageVersion>$(MicrosoftExtensionsDependencyInjectionAbstractionsPackageVersion)</_MicrosoftExtensionsDependencyInjectionAbstractionsPackageVersion>
+    <_MicrosoftExtensionsDependencyInjectionPackageVersion>$(MicrosoftExtensionsDependencyInjectionPackageVersion)</_MicrosoftExtensionsDependencyInjectionPackageVersion>
+    <_MicrosoftExtensionsConfigurationAbstractionsPackageVersion>$(MicrosoftExtensionsConfigurationAbstractionsPackageVersion)</_MicrosoftExtensionsConfigurationAbstractionsPackageVersion>
+    <_MicrosoftExtensionsConfigurationPackageVersion>$(MicrosoftExtensionsConfigurationPackageVersion)</_MicrosoftExtensionsConfigurationPackageVersion>
+    <_MicrosoftExtensionsLoggingAbstractionsPackageVersion>$(MicrosoftExtensionsLoggingAbstractionsPackageVersion)</_MicrosoftExtensionsLoggingAbstractionsPackageVersion>
+    <_MicrosoftExtensionsLoggingPackageVersion>$(MicrosoftExtensionsLoggingPackageVersion)</_MicrosoftExtensionsLoggingPackageVersion>
+    <_MicrosoftExtensionsLoggingConsolePackageVersion>$(MicrosoftExtensionsLoggingConsolePackageVersion)</_MicrosoftExtensionsLoggingConsolePackageVersion>
+    <_MicrosoftExtensionsLoggingDebugPackageVersion>$(MicrosoftExtensionsLoggingDebugPackageVersion)</_MicrosoftExtensionsLoggingDebugPackageVersion>
+    <_MicrosoftExtensionsFileProvidersAbstractionsPackageVersion>$(MicrosoftExtensionsFileProvidersAbstractionsPackageVersion)</_MicrosoftExtensionsFileProvidersAbstractionsPackageVersion>
+    <_MicrosoftExtensionsFileProvidersEmbeddedPackageVersion>$(MicrosoftExtensionsFileProvidersEmbeddedPackageVersion)</_MicrosoftExtensionsFileProvidersEmbeddedPackageVersion>
```

Comment:
Can we just use the properties like `$(MicrosoftExtensionsFileProvidersEmbeddedPackageVersion)` directly and not make new properties?

Reply:
I was wondering this too. For the non-net6 builds we need to use different versions for some packages, so we need _some_ strategy to override them. We could do that by just overriding the versions if not-net6 and using the same variables.

Reply:
what if we stop using maestro and don't a version.props? that's why i created variables for each

Reply:
if i $(MicrosoftExtensionsFileProvidersEmbeddedPackageVersion) for non net6 isn't also the order important? i m not a expert of msbuild so not sure tbh